e2e/config.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
             },
             {
                 expectedPrompt: /Do you wish to use template extensibility?/i,
-                response: '2\n'
+                response: '1\n'
```

Comment:
The order for "Yes" and "No" seems to have flipped in the generator prompts causing incorrect apps from being generated.
This causes the project validation step to fail.

I'll create a follow-up story/PR to replace the entire app generation flow to use the new app config json in the generator command. That eliminates the need to manually stream responses like we do here.

Leaving it as flipped inputs for now is fine.